walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,67 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredOptionalA() {
```

Comment:
What's the difference between everyAttendeeIsConsideredOptionalA() and B? Could you indicate the difference in the test name?